bin/mpdevil

```diff
@@ -3360,7 +3360,7 @@ class AudioFormat(Gtk.Box):
 		self._client=client
 		self._settings=settings
 		self._top_label=Gtk.Label(xalign=1)
-		self._brate_label=Gtk.Label(xalign=1, width_chars=5)
+		self._brate_label=Gtk.Label(xalign=1, width_chars=8)
```

Comment:
8 is definitely not the right choice. I originally set it to 5 to have enough space for 5 digit long bitrates. Since `_bitrate_label` is now also displaying " kb/s • " it should be 5+8=13. But I think a better solution which doesn't waste that much space would be adding a third label `_file_type_label` which only displays the file type. `_top_label` would then be free just for the separating " kb/s • " respectively " kb/s". Maybe the name of the label could be changed to `_separator_label`.